sonar-orchestrator/src/main/java/com/sonar/orchestrator/build/ScannerForMSBuild.java

```diff
@@ -135,13 +138,29 @@ public boolean isDebugLogs() {
   public ScannerForMSBuild setScannerVersion(String s) {
     checkArgument(!isEmpty(s), "version must be set");
     if (s.equals(LATEST_RELEASE)) {
-      s = gitHub.getLatestScannerReleaseVersion();
+      try {
+        s = gitHub.getLatestScannerReleaseVersion();
+      } catch (IllegalStateException e) {
```

Comment:
I would add a log here:

"Failed, to retrieve latest version from github, falling back to repox.jfrog.io"

It will help with us investigating in the future if something happens here before we ask something from the orchestrator owners.

Reply:
I think this one is okay: 

> Failed to retrieve latest version from GitHub, using version from Artifactory